README.md

```diff
@@ -43,22 +67,6 @@ Crack **sha1** hashes from file
     
     ./dehash -type sha1 -file ./hashes.txt
 
-## Get elapsed time
```

Comment:
The command should be left, it works fine, if there are compatibility problems - write, I will try to fix it!

Reply:
There is no time command on Windows, that would be a workaround..